src/main/java/ch/bfh/bti7081/model/User.java

```diff
@@ -32,6 +32,12 @@
     @Column(name = PREFIX + "PASSWORD", length = 512)
     private String password;
 
+    /*
+    permission levels and roles:
+    1 user (logged in)
+    2 expert
+    3 moderator
```

Comment:
Naja, bisher hatten wir in allen Dokumenten 4... (Administrator auch noch)

Reply:
weisst du, wo wir das niedergeschrieben haben? Ich konnte nur die vier aus dem Task 1 finden:

- nicht angemeldeter User
- User
- Experte
- Moderator (wäre doch Administrator?)